Client/Models/Client.cs

```diff
@@ -37,17 +39,28 @@ public sealed class Client
 
         public static Client Instance { get { return lazy.Value; } }
         public string ClientName { get; set; }
-        public int Id { get; set; }
-        public List<Group> Groups { get; set; }
+        public int Id
+        {
+            get
+            {
+                return Id;
+            }
+            set
+            {
+                ApplicationData.Current.LocalSettings.Values["clientId"] = value;
+            }
+        }
```

Comment:
ClientID is set when the client logged in successfully. We don't use sessions so this code isn't used.